driver/controller.go

```diff
@@ -151,6 +154,9 @@ func (d *Driver) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequest)
 		Permanently: true,
 	}
 
+	// Serialize deleting disks
+	defer ovc.ReleaseLock(0)
+	ovc.GetLock(0)
```

Comment:
Doesn't this boil down to a plain global mutex?

Reply:
Yes.

Reply:
Then why not use a plain `sync.Mutex` instead?